src/components/results/sensitivityAnalysis/sensitivity-analysis-contants.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
'contants' in the name of the source file is incorrect. Seems it should be 'content', as for
src/components/diagrams/singleLineDiagram/single-line-diagram-content.js
src/components/diagrams/networkAreaDiagram/network-area-diagram-content.js
src/components/dialogs/commons/modification-dialog-content.js

Reply:
done, thank you